tests/unit/python/foglamp/plugins/south/http_south/test_http_south.py

```diff
@@ -0,0 +1,385 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""Unit test for foglamp.plugins.south.http_south.http_south"""
+import copy
+import json
+import pytest
+import asyncio
+from aiohttp import web
+import aiohttp.web_exceptions
+from unittest import mock
+from unittest.mock import call, patch
+from aiohttp.test_utils import make_mocked_request
+from aiohttp.streams import StreamReader
+from multidict import CIMultiDict
+from foglamp.plugins.south.http_south import http_south
+from foglamp.plugins.south.http_south.http_south import HttpSouthIngest, Ingest, _DEFAULT_CONFIG as config
```

Comment:
Constant variable imported as non constant

Reply:
this is fine for test, I believe!